src/Profile.php

```diff
@@ -66,6 +66,8 @@ public static function fromUuid($uuid, ApiClient $apiClient = null)
             }
         }
 
+        $profile->history = $apiClient->uuidHistory($uuid);
```

Comment:
You also need to add on line 24:

    /** @var string[] */
    public $history;